### PR TITLE
Update iq.m3u

### LINKED ISSUE #3640: Add: Gali Kurdistan TV & Speda TV & Sterk TV
**Channel Name:** Gali Kurdistan
**Country:** Kurdistan, Iraq
**Language:** Kurdish
**Website:** https://www.gksat.tv/
**Stream URL (optional):** http://gkhd.live/
**Notes (optional):** xxx

**Channel Name:** Speda TV
**Country:** Kurdistan, Iraq
**Language:** Kurdish
**Website:** http://speda.net/index.php/ku/
**Stream URL (optional):** http://speda.net/speda.html
**Notes (optional):** xxx

**Channel Name:** SterkTV
**Country:** Kurdistan
**Language:** Kurdish
**Website:** https://sterktv.net/
**Stream URL (optional):** https://sterktv.net/live/
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by Sphinxroot**
close #3640

### CODE REVIEW

File: channels/iq.m3u [hunk 1]
```diff
@@ -105,6 +107,10 @@ https://5aa8c7549bfa9.streamlock.net/liveTrans/ngrp:SHOW1_all/playlist.m3u8
 http://176.10.117.18:8000/play/a006/index.m3u8
 #EXTINF:-1 tvg-id="KurdSatHD.iq" tvg-name="KurdSat HD" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://kurdmaxcms.com/asset/image/footer_logo/show_2.png" group-title="",KurdSat HD
 http://ikomg2.s.llnwi.net/kurdsathd/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/c5ACcvv.jpg" group-title="News",Kurdsatnews
+https://ikomg2.mmdlive.lldns.net/ikomg2/107b7df8f5444d778f349100739a09cd/manifest.m3u8
```
Comment (LaneSh4d0w) [inline channels/iq.m3u:111]: alternative link would be http://ikomg2.s.llnwi.net/kurdsatnewshd/playlist.m3u8 if you want to add it.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by diaqad** [comment]
Can you update om kurdish category also please?

**Comment by LaneSh4d0w** [comment]
The bot will automatically do it tomorrow.

**Comment by LaneSh4d0w** [comment]
However, if you *really* want the channels, you can simply take the m3u file linked to those changes at https://github.com/iptv-org/iptv/raw/8a6e34f25e56cf781ed67c869bbe197d00eef61f/channels/iq.m3u

**Comment by diaqad** [comment]
Nice, thank you. you have added these, hope everyone is in the Kurdish category:

Gali Kurdistan tv.
Sterk tv.
Speda tv.
Netv tv.
KurdsatNews.

**Comment by LaneSh4d0w** [comment]
On verification, yes the mentioned channels are all in the Kurdish category.

**Comment by diaqad** [comment]
@LaneSh4d0w thank you!

**Pull request merged**